beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -451,3 +452,78 @@ func (s *Server) GetAttestationData(w http.ResponseWriter, r *http.Request) {
 	}
 	http2.WriteJson(w, response)
 }
+
+// ProduceSyncCommitteeContribution requests that the beacon node produce a sync committee contribution.
+func (s *Server) ProduceSyncCommitteeContribution(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "validator.ProduceSyncCommitteeContribution")
+	defer span.End()
+
+	subIndex := r.URL.Query().Get("subcommittee_index")
+	index, valid := shared.ValidateUint(w, "Subcommittee Index", subIndex)
+	if !valid {
+		return
```

Comment:
no error written here to the http response?

Reply:
The function called there `ValidateUint` actually returns it's own error if the query parameter is missing. You can see this being checked that the error is returned correctly in the tests:
```
require.ErrorContains(t, "Subcommittee Index is required", errors.New(writer.Body.String()))
```
And in the case of `produceSyncCommitteeContribution` we write multiple http errors based on the circumstance.